blocks/components/editable/index.js

```diff
@@ -57,6 +59,28 @@ export default class Editable extends wp.element.Component {
 		this.props.onChange( value );
 	}
 
+	onKeyDown( event ) {
+		if ( ! this.props.tagName && event.keyCode === 13 ) {
+			// Wait for the event to propagate
+			setTimeout( () => {
+				// Getting the content before and after the cursor
+				this.editor.selection.getStart();
```

Comment:
Problem with a `setTimeout` is we can't be guaranteed that the component didn't unmount in the last tick, meaning due to the behavior of the function `ref`, `this.editor` could potentially be `null`.

>React will call the `ref` callback with the DOM element when the component mounts, and call it with `null` when it unmounts.

https://facebook.github.io/react/docs/refs-and-the-dom.html